frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/StartHandler.java

```diff
@@ -238,9 +241,42 @@ public APIGatewayProxyResponseEvent handleRequest(
 
             boolean isBlockedForReauth = false;
             if (configurationService.isAuthenticationAttemptsServiceEnabled() && reauthenticate) {
```

Comment:
This bit of feature flag checking and reauthenticate check can probably be removed now.

The question of "can the user start a journey" is answered by `permissionDecisionManager.canStartJourney`.